CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 Changes
 =======
 
+# 0.3.1 / 11-03-2015
```

Comment:
Since we'll probably add more things to this version bump, let's say this instead, `0.3.1 / Unreleased`